index.js

```diff
@@ -24,67 +26,81 @@ function getRandomNumber(min, max) {
 
 function checkGuess() {
   // Get value from guess input element
-  const guess = parseInt(guessInput.value, 10);
+  let guess = parseInt(guessInput.value, 10);
```

Comment:
This one can stay `const`, I think. It's generally preferable to use `const` because one source of bugs is making accidental changes to values. `const` prevents that.